safe-global/safe-homepage
# Fetch ecosystem data on build time

## What it solves

Fetches the ecosystem data on build time via `getStaticProps`

This could be combined with `next-pwa` to add offline support if we want.

## How to test

1. Build, export and run the app locally e.g. via npx serve
2. Open the Ecosystem page
3. See the projects are visible
